### PR TITLE
Added setHMAC method for iOS

### PR DESCRIPTION
**Pull request by secobarbital**
Required for Secure Mode

### CODE REVIEW

File: iOS/IntercomWrapper.m [hunk 1]
```diff
@@ -156,5 +156,12 @@ @implementation IntercomWrapper
     callback(@[[NSNull null]]);
 };
 
+// Available as NativeModules.IntercomWrapper.setHMAC
+RCT_EXPORT_METHOD(setHMAC:(NSString*)hmac data:(NSString*)data callback:(RCTResponseSenderBlock)callback) {
+    NSLog(@"setHMAC with %@", hmac);
```
Comment (browniefed) [inline iOS/IntercomWrapper.m:161]: Can you remove the log

Comment (secobarbital) [inline iOS/IntercomWrapper.m:161]: Certainly! I didn't like them but wanted to be consistent with the existing methods.

Comment (browniefed) [inline iOS/IntercomWrapper.m:161]: Ah thanks, I'll need to go remove them.

**Comment by browniefed** [comment]
Thanks, I'll get a new version out shortly

**Pull request merged**